python/foglamp/services/common/microservice.py

```diff
@@ -92,12 +92,19 @@ def _get_service_registration_payload(self):
             }
         return service_registration_payload
 
+    async def shutdown(self, request=None):
```

Comment:
request is required to make it a valid web endpoint for aiohttp web server instance